### PR TITLE
Release/2.2.0

### PR DESCRIPTION
**Pull request by stefaniacardenas**
### Added
- Public: Added support for internationalisation. The SDK can now be displayed in Spanish by adding `{language: 'es'}` to the initialization options. It can also be displayed in a custom language by passing an object containing the custom phrases and the locale. If `language` is not present or the wrong locale tag is provided, the language locale will default to `en`.
- Public: Added support for Spanish language on the SMS body.
- Public: Added webcam support on Safari and IE Edge.

### Changed
- UI: If the webcam is facing the user it will be mirrored

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [x] Have new automated tests been implemented?
- [x] Have new manual tests been written down?
- [x] Have tests passed locally?
- [x] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-309-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
 # Changelog
```
Comment (dawiddylowicz) [inline CHANGELOG.md]: Don't forget to add `[2.2.0]: https://github.com/onfido/onfido-sdk-ui/compare/2.1.0...2.2.0` at the end of CHANGELOG.md

File: MANUAL_REGRESSION.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
+# Manual Regression Tests for JS SDK
```
Comment (dawiddylowicz) [inline MANUAL_REGRESSION.md]: Well done on the new tests!

File: MANUAL_REGRESSION.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+2. Then the title on the welcome screen should be 'A custom string'
+3. All the other strings should be in Spanish
+
+##### 11. Overriding strings for a supported language on mobile
+0. Go to latest JsFiddle
+1. Add the following options to the initialisation params:
+  ```javascript
+  language: {
+    locale: 'es',
+    localePhrases: {'capture.passport.front.title': 'A custom string'}
```
Comment (dawiddylowicz) [inline MANUAL_REGRESSION.md]: According to README.md, should it be `mobilePhrases`?

Comment (stefaniacardenas) [reply MANUAL_REGRESSION.md]: Good spot!

File: README.md [hunk 4]
```diff
@@ -214,6 +215,34 @@ A number of options are available to allow you to customise the SDK:
...
+  The object should include the following keys:
+    - `locale` (optional) : A locale tag. In order to partially customise the strings of a supported language (ie. Spanish), you will need to pass the locale tag. For missing keys, a warning and an array containing the missing keys will be returned on the console. The values for the missing keys will be displayed in the language specified within the locale tag if supported, otherwise they will be displayed in English. The locale tag is also used to override the language of the SMS body for the cross device feature. This feature is owned by Onfido and is currently only supporting English and Spanish.
+    - `phrases` (required) : An object containing the keys you want to override and the new values. The keys can be found in `/src/locales/en.js`. They can be passed as a nested object or as a string using the dot notation for nested values. See the examples below.
+    - `mobilePhrases` (optional) : An object containing the keys you want to override and the new values. The values specified within this object are only visible on mobile devices. Please refer to `/locales/mobilePhrases/en.js`.
```
Comment (rfreitas) [inline README.md]: is it missing `src` from the path?

Comment (stefaniacardenas) [reply README.md]: Yes!

File: MANUAL_REGRESSION.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+  }
+  ```
+2. Select passport on the document selector screen
+3. Choose the cross device flow and send an SMS to your mobile
+4. The SMS should be in Spanish
+5. When you open the link on your mobile device, the title on the cross device client should be 'A custom string'
+6. All the other strings should be in Spanish
+
+##### 12. Upload a document in PDF format
+(on Google Chrome, Firefox, Safari, IE11 and Microsoft Edge browsers)
```
Comment (dawiddylowicz) [inline MANUAL_REGRESSION.md]: We have an automated test for this scenario. It runs on Chrome by default (also should be able to run on Firefox - but we don't do it on CI yet) so I think we could at least remove Chrome from this test

Comment (stefaniacardenas) [reply MANUAL_REGRESSION.md]: Yes, I will remove Chrome then but I think we should still perform manual tests on other browsers.

Comment (dawiddylowicz) [reply MANUAL_REGRESSION.md]: Yes test should stay for other browsers

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-309-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://309-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**